api/WebTransport.json

```diff
@@ -384,7 +418,9 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "114"
+              "version_added": "114",
+              "partial_implementation": true,
+              "notes": "Returns a <code>WritableStream</code> instead of a <code>WebTransportSendStream</code>."
```

Comment:
Note, the spec says `WebTransportSendStream` but FF is the only implementer, and it returns writable stream.